chapters/openra/index.md

```diff
@@ -0,0 +1,477 @@
+
+<!-- michielha: maybe chuck in a screenshot in here to visualize what you just said -->
+
+## The conception of OpenRA
+
+OpenRA, where RA is an abbreviation of *Command & Conquer: Red Alert* --the best-received game of the Command & Conquer series-- is an open source project that aims for the reimplementation of the game engine featured in a number of Command & Conquer games.
+Besides the bare engine, the project features a number of so-called *mods* that can be run on the engine, with the mods mimicking the gameplay of individual games.
+As of March 2015, there are playable mods for the Red Alert, Dune 2000, and Tiberium Dawn games.
+A mod for Red Alert 2 is in active development.
+
+The OpenRA project was started by Chris Forbes in June 2007, presumably out of nostalgia for the old games.
+The project lay relatively dormant for about two years.
+Then suddenly, in October 2009, the project quickly gained a number of much-needed contributors, providing the spark that started a development effort that today is stronger than ever.
+At the time of writing, the project has amassed 15220 commits by 159 individuals, including some contributions by the authors of this chapter.
+
+# Stakeholder Analysis
+
+Stakeholders within OpenRA work a little bit different than in a commercial(ly backed) product.
+As virtually no money is involved, most contributions come from a very small set of people that take many tasks upon themselves.
+
+Within the OpenRA project, the contributors aren't really concerned with a (materialistic) return on investment.
+There is however a thriving community of people concerning themselves with the planning of the project: creating road maps for the project, categorizing issues and reporting bugs.
+As we can see from the [issue](https://github.com/openra/openra/issues) and [pull request](https://github.com/openra/openra/pulls) page, most issue and milestone management is done by [@Mailaender](https://github.com/Mailaender), [@pchote](https://github.com/pchote) and [@obrakmann](https://github.com/obrakmann), who also have been the top contributors over the last twelve months.
+
+A summary of the project's stakeholders can be found in [Table 1](#t1).
+
+<a name="t1"></a>
+
+ Type | Entities |
+|---|---|
+| Developers | [@pchote](https://github.com/pchote) , [@Mailaender](https://github.com/Mailaender), [@obrakmann](https://github.com/obrakmann), [@chrisforbes](https://github.com/chrisforbes) |
+| Users | hundreds of players around the world, [ModDB](http://www.moddb.com/games/openra) |
+| Suppliers | .NET/Mono, [Westwood](http://nl.wikipedia.org/wiki/Westwood_Studios), [Electronic Arts](http://www.ea.com/) |
+| Assessors | Preventing copyright infringement done by [@pchote](..) and [@mailaender](..) in https://github.com/openra/openra/issues?q=legal |
```

Comment:
They do. However, for you convenience, I've added them anyway :camel: :dromedary_camel: